libsrc/telestrat/tgi/telestrat-228-200-3.s

```diff
@@ -297,7 +297,13 @@ LINE:
         lda   Y2
         sta   HRS4
         
-        lda   #$ff
+        lda   #$00
+        sta   HRS1+1
+        sta   HRS2+1
+        sta   HRS3+1
+        sta   HRS4+1        
```

Comment:
Don't know anything about the Telestrat OS, but if this is needed here shouldn't also be needed at SETPIXEL?

Reply:
I tested SETPIXEL, and the problem only appears with LINE